Source/Core/Core/HW/SI/SI.h

```diff
@@ -56,6 +56,7 @@ class SerialInterfaceManager
   void RemoveEvent(int device_number);
 
   void UpdateDevices();
+  u32 GetInLength(void) const { return m_com_csr.INLNGTH; }
```

Comment:
```suggestion
  u32 GetInLength() const { return m_com_csr.INLNGTH; }
```

(It does work fine as it is, but it's unnecessary. This isn't C.)